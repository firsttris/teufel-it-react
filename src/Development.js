import React, { Component } from 'react';
import App from './App';
import MuchCode from './assets/images/much_code.jpg';
import CodeColor from './assets/images/code_color.jpg';
import PrinciplesJson from './assets/json/development/principles.json'
import PortfolioJson from './assets/json/development/portfolio.json'

class Development extends Component {
    render() {
        return (
            <div>
                <App image1={MuchCode} image2={CodeColor} image3={MuchCode} portfolioJson={PortfolioJson} principlesJson={PrinciplesJson}></App>
            </div>
        );
    }
}

export default Development;