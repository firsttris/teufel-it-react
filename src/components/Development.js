import React, {Component} from 'react';
import Parallax from './Parallax';
import MuchCode from './../assets/images/much_code.jpg';
import CodeColor from './../assets/images/code_color.jpg';
import development from '../assets/pages/development.json'
import {Helmet} from "react-helmet";

export default class Development extends Component {
    render() {
        return (
            <div>
                <Helmet>
                    <title>Teufel IT, Software ist unser Handwerk</title>
                    <meta charSet="utf-8" />
                    <meta name="description" content="Teufel IT, Softwareentwicklung, JavaScript, NodeJs, Integration" />
                    <link rel="canonical" href="http://teufel-it.de" />
                </Helmet>
                <Parallax image1={MuchCode} image2={CodeColor} image3={MuchCode} json={development}/>
            </div>
        );
    }
}