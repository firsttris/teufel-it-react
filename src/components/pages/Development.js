import React, {Component} from 'react';
import Parallax from './../Parallax';
import OldComputer from './../../../public/images/oldComputer2.jpg';
import MuchCode from './../../../public/images/much_code.jpg';
import DevelopmentJson from './../../../public/pages/development.json';
import {Helmet} from "react-helmet";

export default class Development extends Component {
    render() {
        return (
            <div>
                <Helmet>
                    <title>Teufel IT, Software ist unser Handwerk</title>
                    <meta charSet="utf-8" />
                    <meta name="description" content="Teufel IT, Softwareentwicklung, JavaScript, NodeJs, Integration" />
                    <link rel="canonical" href="http://teufel-it.de" />
                </Helmet>
                <Parallax image1={OldComputer} image2={MuchCode} image3={OldComputer} json={DevelopmentJson}/>
            </div>
        );
    }
}