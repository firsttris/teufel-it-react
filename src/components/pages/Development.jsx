import React, { Component } from 'react';
import Parallax from './Parallax.jsx';
import OldComputer from 'images/oldComputer2.jpg';
import MuchCode from 'images/much_code.jpg';
import DevelopmentJson from 'json/pages/development.json';
import { Helmet } from 'react-helmet';

class Development extends Component {
  render() {
    return (
      <div>
        <Helmet>
          <title>Teufel IT, Software ist unser Handwerk</title>
          <meta charSet="utf-8" />
          <meta
            name="description"
            content="Teufel IT, Softwareentwicklung, JavaScript, NodeJs, Integration"
          />
          <link rel="canonical" href="http://teufel-it.de" />
        </Helmet>
        <Parallax
          image1={OldComputer}
          image2={MuchCode}
          image3={OldComputer}
          json={DevelopmentJson}
        />
      </div>
    );
  }
}

export default Development;
