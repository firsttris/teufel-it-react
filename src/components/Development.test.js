import React from 'react';
import Development from './Development';
import renderer from 'react-test-renderer';

test('Display Component', () => {
  const component = renderer.create(
    <Development></Development>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});