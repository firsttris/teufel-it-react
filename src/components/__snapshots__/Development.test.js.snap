// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component 1`] = `
<div>
  <div
    className="container-fluid"
    style={
      Object {
        "backgroundImage": "url(lines.png)",
        "textAlign": "center",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "padding": 0,
          }
        }
      >
        <div
          style={
            Object {
              "backgroundAttachment": "fixed",
              "backgroundImage": "url(much_code.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
              "height": "550px",
              "minHeight": "400px",
              "opacity": "0.65",
              "position": "relative",
            }
          }
        />
      </div>
    </div>
    <div
      className="row py-3"
    >
      <div
        style={
          Object {
            "position": "absolute",
            "top": "200px",
            "width": "100%",
          }
        }
      >
        <div
          className="p-3"
          style={
            Object {
              "backgroundColor": "#4B4B4B",
              "display": "inline-block",
            }
          }
        >
          <div
            style={
              Object {
                "color": "#f00",
                "fontSize": "35px",
                "letterSpacing": "8px",
              }
            }
          >
            [ teufel it ]
          </div>
          <div
            className="pt-3"
            style={
              Object {
                "color": "white",
                "fontSize": "15px",
                "letterSpacing": "5px",
              }
            }
          >
            Software ist unser Handwerk
          </div>
        </div>
      </div>
    </div>
    <div
      className="row py-3"
    >
      <div
        className="col"
      >
        <div
          className="text-uppercase"
          style={
            Object {
              "fontSize": "30px",
              "letterSpacing": "10px",
            }
          }
        >
          portfolio
        </div>
        <p>
          Wir sind zuverlässiger Partner bei
        </p>
      </div>
    </div>
    <div
      className="row py-3"
    >
      <div
        className="col-xs-12 col-md-6 col-lg-3"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-github"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              software
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Entwicklung von Nativen-, Web- und Cross-Platform Anwendungen.
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-6 col-lg-3"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-mobile"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              mobile
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Mobile Entwicklung für iOS und Android.
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-6 col-lg-3"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-code-fork"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              services
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Integration und Migration von Business Services.
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-6 col-lg-3"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "height": "250px",
            "perspective": "800px",
          }
        }
      >
        <div
          style={
            Object {
              "backfaceVisibility": "hidden",
              "transform": "rotateY(0deg)",
              "transformStyle": "preserve-3d",
              "transition": "transform 1s linear",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "width": "100%",
              }
            }
          >
            <i
              aria-hidden="true"
              className="fa fa-5x fa-graduation-cap"
              style={
                Object {
                  "color": "#f00",
                  "fontSize": "10em",
                }
              }
            />
            <div
              className="text-uppercase"
              style={
                Object {
                  "fontSize": "1.5em",
                }
              }
            >
              training
            </div>
          </div>
          <div
            style={
              Object {
                "backfaceVisibility": "hidden",
                "position": "absolute",
                "top": "0",
                "transform": "rotateY(180deg)",
                "width": "100%",
              }
            }
          >
            <div
              className="p-2"
              style={
                Object {
                  "alignItems": "center",
                  "border": "1px solid #f00",
                  "borderRadius": "5px",
                  "display": "flex",
                  "height": "175px",
                  "margin": "auto",
                  "width": "50%",
                }
              }
            >
              Wir teilen unser Wissen, neue Konzepte und Methoden.
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "padding": 0,
          }
        }
      >
        <div
          style={
            Object {
              "backgroundAttachment": "fixed",
              "backgroundImage": "url(code_color.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
              "height": "350px",
              "minHeight": "400px",
              "opacity": "0.65",
              "position": "relative",
            }
          }
        />
      </div>
    </div>
    <div
      className="row py-3"
      style={
        Object {
          "backgroundImage": "url(paper.png)",
          "color": "white",
        }
      }
    >
      <div
        className="col"
      >
        <div
          className="text-uppercase"
          style={
            Object {
              "fontSize": "30px",
              "letterSpacing": "10px",
            }
          }
        >
          prinzipien
        </div>
        <p>
          Wir arbeiten auf Basis von bewährten Konzepten
        </p>
      </div>
    </div>
    <div
      className="row py-3"
      style={
        Object {
          "backgroundImage": "url(paper.png)",
          "color": "white",
        }
      }
    >
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-free-code-camp"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            tdd
          </span>
        </div>
        <p>
          Wir testen schon ab der ersten Zeile
        </p>
      </div>
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-users"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            pair programming
          </span>
        </div>
        <p>
          Lösungen im Team sind langfristig nachhaltiger
        </p>
      </div>
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-code"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            clean code
          </span>
        </div>
        <p>
          Prinzipien und Praktiken zu mehr Qualität
        </p>
      </div>
      <div
        className="py-2 col-xs-12 col-md-6 col-lg-6"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <div
          style={
            Object {
              "alignItems": "center",
              "display": "inline-flex",
            }
          }
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-refresh"
            style={
              Object {
                "color": "#f00",
                "transition": "color 2s linear 1s",
              }
            }
          />
          <span
            className="text-uppercase"
            style={
              Object {
                "fontSize": "20px",
                "marginLeft": "10px",
              }
            }
          >
            agile
          </span>
        </div>
        <p>
          Wir lösen komplexe Herausforderungen in Iterationen
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "padding": 0,
          }
        }
      >
        <div
          style={
            Object {
              "backgroundAttachment": "fixed",
              "backgroundImage": "url(much_code.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
              "height": "350px",
              "minHeight": "400px",
              "opacity": "0.65",
              "position": "relative",
            }
          }
        />
      </div>
    </div>
    <div
      className="row pt-3"
    >
      <div
        className="col"
      >
        <div
          className="text-uppercase"
          style={
            Object {
              "fontSize": "30px",
              "letterSpacing": "10px",
            }
          }
        >
          Kontakt
        </div>
        <p>
          Kontaktieren Sie mich über
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col"
        style={
          Object {
            "margin": "0 auto",
          }
        }
      >
        <a
          aria-hidden="true"
          className="fa fa-2x fa-xing-square"
          href="https://www.xing.com/profile/Tristan_Teufel"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#f00",
              "margin": "10px",
              "textDecoration": "none",
            }
          }
          target="_blank"
        >
           
        </a>
        <a
          aria-hidden="true"
          className="fa fa-2x fa-linkedin-square"
          href="https://de.linkedin.com/pub/tristan-teufel/8a/30a/b5"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#f00",
              "margin": "10px",
              "textDecoration": "none",
            }
          }
          target="_blank"
        >
           
        </a>
        <a
          aria-hidden="true"
          className="fa fa-2x fa-github"
          href="https://github.com/firsttris"
          rel="noopener noreferrer"
          style={
            Object {
              "color": "#f00",
              "margin": "10px",
              "textDecoration": "none",
            }
          }
          target="_blank"
        >
           
        </a>
      </div>
    </div>
    <div
      className="row pb-3"
    >
      <div
        className="col"
        style={
          Object {
            "margin": "0 auto",
          }
        }
      >
        <a
          className="btn btn-secondary"
          href="tel:+49 176 45744166"
          rel="noopener noreferrer"
          style={
            Object {
              "margin": "10px",
            }
          }
          target="_blank"
        >
          +49 176 45744166
        </a>
        <a
          className="btn btn-secondary"
          href="mailto:info@teufel-it.de"
          rel="noopener noreferrer"
          style={
            Object {
              "margin": "10px",
            }
          }
          target="_blank"
        >
          info@teufel-it.de
        </a>
      </div>
    </div>
    <div
      className="row py-3"
      style={
        Object {
          "backgroundImage": "url(paper.png)",
          "color": "white",
        }
      }
    >
      <div
        className="col-xs-12 col-md-4 col-lg-4"
      >
        <div>
          Inhaber:
        </div>
        <div>
          Tristan Teufel
        </div>
        <div>
          Werkstr.10a
        </div>
        <div>
          D-77815 Bühl
        </div>
      </div>
      <div
        className="col-xs-12 col-md-4 col-lg-4"
      >
        <span>
          Build with 
          <a
            href="https://facebook.github.io/react/"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            <b>
              React
            </b>
          </a>
        </span>
        <img
          alt="logo"
          className="App-logo"
          src="logo.svg"
        />
        <div>
          <div>
            Quellen für Bilder und Icons:
          </div>
          <a
            href="http://unsplash.com"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            unsplash.com 
          </a>
          <a
            href="http://shutterstock.com"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            shutterstock.com 
          </a>
          <a
            href="http://fontawesome.io"
            rel="noopener noreferrer"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
            target="_blank"
          >
            fontawesome.io
          </a>
        </div>
      </div>
      <div
        className="col-xs-12 col-md-4 col-lg-4"
      >
        <div>
          Kontakt:
        </div>
        <div>
          <a
            href="tel:+4917645744166"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
          >
            +49 (0) 176 457 44 1 66
          </a>
        </div>
        <div>
          <a
            href="mailto:info@teufel-it.de"
            style={
              Object {
                "color": "#61DAFB",
                "textDecoration": "none",
              }
            }
          >
            info@teufel-it.de
          </a>
        </div>
        <div>
          Finanzamt Baden-Baden 36216/14616
        </div>
      </div>
    </div>
  </div>
</div>
`;
